### ISSUE TITLE
[Debugger] When creating a method the focus should be in the code pane

### ISSUE THREAD

**Issue by tesonep** [pharo-spec/NewTools#123]
When a method is created in the debugger, after selecting the package.
The keyboard focus should be in the code pane and not in the stack frames list.
